src/main/java/com/gitblit/manager/FederationManager.java

```diff
@@ -363,6 +363,10 @@ public boolean accept(File file) {
 							&& file.getName().toLowerCase().endsWith(Constants.PROPOSAL_EXT);
 				}
 			});
+			if (files == null) {
```

Comment:
Yes but this returns the empty initialized list before files(null) can throw a NPE. Or Not?